src/os.c

```diff
@@ -492,18 +492,18 @@ static const JSCFunctionListEntry tjs_os_funcs[] = {
     TJS_CFUNC_DEF("_envKeys", 0, tjs_envKeys),
     TJS_CFUNC_DEF("_environ", 0, tjs_environ),
     TJS_CFUNC_DEF("chdir", 1, tjs_chdir),
-    TJS_CFUNC_DEF("cwd", 0, tjs_cwd),
-    TJS_CFUNC_DEF("homedir", 0, tjs_homedir),
-    TJS_CFUNC_DEF("tmpdir", 0, tjs_tmpdir),
     TJS_CFUNC_DEF("random", 3, tjs_random),
     TJS_CFUNC_DEF("cpuInfo", 0, tjs_cpu_info),
     TJS_CFUNC_DEF("loadavg", 0, tjs_loadavg),
     TJS_CFUNC_DEF("networkInterfaces", 0, tjs_network_interfaces),
-    TJS_CFUNC_DEF("gethostname", 0, tjs_gethostname),
-    TJS_CFUNC_DEF("getPid", 0, tjs_getpid),
-    TJS_CFUNC_DEF("getPpid", 0, tjs_getppid),
-    TJS_CFUNC_DEF("userInfo", 0, tjs_userInfo),
     TJS_CFUNC_DEF("availableParallelism", 0, tjs_availableParallelism),
```

Comment:
Personally I would have called it `threads` because it is a bit of a mouthful, but that is just a very subjective opinion :D.

Reply:
I was referring to hardware threads, not software ones.
Like Sockets > Cores > Threads > Pipeline.  
Usually the parallelism of a system is linked to threads since they are the latests stages capable of virtually performing all operations of the instruction set (with many asterisks).  
But yes, if this nomenclature creates ambiguity maybe it is better to leave it as it is :D.